README.md

```diff
@@ -8,34 +8,32 @@ subscribe to notifications for service disruptions.
 ### Requirements
 
 - MBTA API key (get one [here](https://api-dev.mbtace.com))
-  - **Note:** This key must have its version set to `2019-04-05`
+  - **Note:** This key must have its version set to `2019-02-12`
+  - You may need to request an increased rate limit if you get errors during
+    the first app startup; 2000 requests per minute should be enough
 - PostgreSQL 13 (using Homebrew: `brew install postgresql@13`)
+  - You will need Postgres client tools in your PATH; if using Homebrew and you
+    get an error about missing tools, use `brew link postgresql@13`
+- [Google Chrome](https://www.google.com/chrome/)
 - Chromedriver (using Homebrew: `brew cask install chromedriver`)
-- Erlang, Elixir, and Node.js versions specified in `.tool_versions`
-  - Use [`asdf`](https://github.com/asdf-vm/asdf) to install automatically
-    - Note [these extra install steps][nodejs-reqs] for NodeJS plugin
-- [direnv](https://github.com/direnv/direnv) _(optional, but convenient)_
-
-[nodejs-reqs]: https://github.com/asdf-vm/asdf-nodejs#requirements
+- [`asdf`](https://asdf-vm.com/) with plugins: `elixir`, `erlang`, `nodejs`
+- [`direnv`](https://direnv.net/)
```

Comment:
I think we can still consider this optional, maybe also keep the part about `source`ing `.envrc`

Reply:
If you think it's worth mentioning the alternative then I'll leave it in!